packages/cli/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "test": "pnpm run lint && borp --timeout 120000 --concurrency 1",
     "lint": "standard | snazzy",
-    "license": "license-checker --production --onlyAllow 'Apache-2.0;MIT;ISC;BSD-2-Clause;BSD-3-Clause;CC-BY-4.0'"
+    "license": "license-checker --production --onlyAllow 'Apache-2.0;MIT;ISC;BSD-2-Clause;BSD-3-Clause;CC-BY-4.0;Unlicense'"
```

Comment:
I would rather not to, as it would fail most of the corporate license checks. Is there a similar module that solves the same problem?

Reply:
It's easier to write our own

Reply:
replaced with https://www.npmjs.com/package/@watchable/unpromise